apps/console/src/features/server-configurations/models/governance-connectors.ts

```diff
@@ -1,16 +1,16 @@
 /**
- * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ * Copyright (c) 2021, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
DO we need to change the year? Check the other places as well.

Reply:
https://github.com/wso2/identity-apps/pull/3455

Reply:
This is the new header, the year is changed in the new one
https://github.com/wso2/identity-apps/pull/3455